papers/donath/donath-pylira.rst

```diff
@@ -0,0 +1,423 @@
+    setup and show a Chandra as well as Fermi-LAT analysis example.
+
+
+
+.. class:: keywords
+
+   deconvolution, point spread function, poisson, low counts, x-ray, gamma-ray
+
+Introduction
+------------
+Any physical and astronomical imaging process is affected by the limited
+angular resolution of the instrument or telescope. In addition the quality
+of the resulting image is also degraded by background or instrumental
+measurement noise and non-uniform exposure.
+For short wavelengths and associated low intensities
+of the signal, the imaging process consists in recording individual photons arriving from
+a source of interest, often called "events".
+This imaging process is typically for x-ray and gamma-ray telescopes, but is also found
+in magnetic resonance imaging or fluorescence microscopy.
+For each individual photon the incident direction and energy is
+measured. Based on this information the event can be histogramed
+into two dimensional data structures to form an actual image.
+
+Because of the low intensities associated recording of individual
+events, the measured signal follows Poisson statistics. This imposes
+a non-linear relationship between the measured signal and true
+underlying intensity as well as a coupling of the signal with the
+measurement noise. Any statistically correct post-processing
+or reconstruction method of thus requires are
+careful treatment of the Poisson nature of the measured image.
+
+To maximise the scientific use of the data is is often desired
+to correct the degradation introduced by the imaging process.
+Besides correction for non-uniform exposure and background
+noise this also includes the correction for the "blurring"
+introduced by the point spread function (PSF) of the
+instrument, often called "deconvolution". Depending on whether
+the PSF of the instrument is known, one distinguishes between
+the "blind deconvolution" and "non blind deconvolution" process.
+For astronomical observations the PSF can often either be
+simulated, given a model of the detector or inferred
+directly from the data by observing far distant objects,
+which appear as point source to the instrument.
+
+While in other branches of astronomy deconvolution methods are already part
+of the standard analysis, such as the CLEAN algorithm for radio data, this
+is not the case for x-ray and gamma-ray astronomy. As any deconvolution method
+aims to enhance small scale structures in an image it becomes increasingly
+hard to solve for the regime of low signal to noise ratio.
+State of the art de-blurring algorithms based on deep convolutional neural networks
+show exceptional results on conventional image data,
+but they cannot be straightforwardly applied to astronomical counts data,
+because of a lack of ground truth and training data in general.
+
+
+Deconvolution Methods
+---------------------
+
+Richardson-Lucy
++++++++++++++++
+One of the first methods for deconvolution of images with Poisson noise was
```

Comment:
I think original R-L algorithm does not deal with Poisson noise.  In Richardson paper he specifically says that his algorithm is for high noise and includes a random noise as a multiplicative factor at different levels. Here is the Eq. 10 from Richardson paper, where H is the degraded, observed image:
"H'=H(I+ rd), H is the noise-free value, r is the fraction of added noise, and d is the random deviate, normal (0,1)". I guess this is a random value from a Gaussian distribution, not Poisson...

Reply:
The last time I checked I just referred to the Wikipedia reference https://en.wikipedia.org/wiki/Richardson–Lucy_deconvolution#Derivation, which explicitly assumes Poisson statistics per pixel. But I'll check the original papers once more...will leave open for now...